projects/subgraph-wells/src/WellHandler.ts

```diff
@@ -3,88 +3,95 @@ import { loadOrCreateAccount } from "./utils/Account";
 import { emptyBigIntArray, ZERO_BI } from "./utils/Decimals";
 import { recordAddLiquidityEvent, recordRemoveLiquidityEvent, recordRemoveLiquidityOneEvent } from "./utils/Liquidity";
 import { recordSwapEvent } from "./utils/Swap";
-import { checkForSnapshot, incrementWellDeposit, incrementWellSwap, incrementWellWithdraw, loadWell, updateWellLiquidityTokenBalance, updateWellTokenBalances, updateWellVolumes } from "./utils/Well";
+import {
+  checkForSnapshot,
+  incrementWellDeposit,
+  incrementWellSwap,
+  incrementWellWithdraw,
+  loadWell,
+  updateWellLiquidityTokenBalance,
+  updateWellTokenBalances,
+  updateWellVolumes
+} from "./utils/Well";
 
 export function handleAddLiquidity(event: AddLiquidity): void {
+  loadOrCreateAccount(event.transaction.from);
```

Comment:
withdraws and deposits showing 0 for `amountUSD` always (not related to this line just mentioning it via PR review)